sfputil/debug.py

```diff
@@ -0,0 +1,327 @@
+import sys
+import click
+import sonic_platform
+import sonic_platform_base.sonic_sfp.sfputilhelper
+from sonic_platform_base.sfp_base import SfpBase
+from sonic_py_common import multi_asic
+import utilities_common.cli as clicommon
+from utilities_common import platform_sfputil_helper
+from utilities_common.platform_sfputil_helper import (
+    logical_port_name_to_physical_port_list,
+    logical_port_to_physical_port_index,
+    is_rj45_port,
+    is_sfp_present,
+)
+from swsscommon.swsscommon import SonicV2Connector, ConfigDBConnector
+
+# Global variables for platform-specific utilities
+platform_sfputil = None
+platform_chassis = None
+
+EXIT_FAIL = 1  # Exit code for failure
+ERROR_NOT_IMPLEMENTED = 2  # Error code for unimplemented functionality
+
+
+def get_subport_lane_mask(subport, lane_count):
+    """
+    Get the lane mask for the given subport and lane count.
+
+    This method calculates the lane mask based on the subport and lane count.
+
+    Args:
+        subport (int): The subport number to calculate the lane mask for.
+        lane_count (int): The number of lanes per subport.
+
+    Returns:
+        int: The lane mask calculated for the given subport and lane count.
+    """
+    # Calculating the lane mask using bitwise operations.
+    return ((1 << lane_count) - 1) << ((subport - 1) * lane_count)
+
+
+def get_sfp_object(port_name):
```

Comment:
@vdahiya12 there could be other users of this in future so better common utility place?

Reply:
fixed